packages/language-server/test/completion-items-spec.ts

```diff
@@ -244,6 +244,24 @@ describe("the UI5 language assistant Code Completion Services", () => {
     ]);
   });
 
+  it("will get completion values for UI5 experimental class", () => {
+    const xmlSnippet = `<mvc:View 
+                          xmlns:mvc="sap.ui.core.mvc" 
+                          xmlns="sap.m">
+                          <content> <ContentS⇶`;
```

Comment:
why is this snippet request completion on the closing tag?

Reply:
It's not on the closing tag?

Reply:
It's not for closing tag, it's for "ContentSwitcher"

Reply:
ah the indentation confused me